lib/omni_auth/multi_provider/handler.rb

```diff
@@ -2,23 +2,26 @@ module OmniAuth
   module MultiProvider
     class Handler
       attr_reader :path_prefix, :provider_instance_path_regex, :request_path_regex,
-                  :callback_path_regex, :provider_path_prefix,
+                  :callback_path_regex, :callback_postfix,
                   :identity_provider_options_generator
 
       def initialize(path_prefix:,
                      identity_provider_id_regex:,
+                     **options,
```

Comment:
We can simplify this a bit by declaring the named parameter (i.e.`callback_suffix: 'callback'`) in the function declaration. This will eliminate the need for `DEFAULT_OPTIONS`, merging, etc.